core/typescript/src/lib/metadata.ts

```diff
@@ -169,9 +189,10 @@ export interface ConstructsInfo {
    */
   readonly schema: string;
   /**
-   * Information about network.
+   * Information about optional network.
+   * Missing this field implies that the package does NOT support customisable virtual network.
    */
-  readonly network: NetworkInfo;
+  readonly network?: NetworkInfo;
```

Comment:
VPC in some public cloud providers could be out of reach, or a package could choose to disallow it